jax/numpy/lax_numpy.py

```diff
@@ -545,6 +545,28 @@ def exp2(x):
   return lax.exp(lax.mul(lax.log(_constant_like(x, 2)), x))
 
 
+@_wraps(onp.signbit)
+def signbit(x):
+  x, = _promote_shapes("signbits", x)
+
+  if issubdtype(_dtype(x), onp.integer):
+    x = lax.convert_element_type(x, onp.float64)
```

Comment:
This seems unnecessarily heavyweight for integers (casting to a float). For integer types, can't we just do something like:
```
return lax.lt(x, _constant_like(x, 0))
```